triage/network_info.go

```diff
@@ -17,8 +17,8 @@ func printNetworkInfo() {
 }
 
 func printHostname() string {
-	var hostname, name_err = os.Hostname()
-	if name_err == nil {
+	var hostname, nameErr = os.Hostname()
+	if nameErr == nil {
```

Comment:
Shouldn't the logic be the inverse here? 
```suggestion
	if nameErr != nil {
	  return ""
	}
```
I.e. it shouldn't be returning `hostname` if there was an error.
Since this method is `printHostname`, arguably it shouldn't return anything. Consider making a `getHostname` and have `printHostname` call that?

Reply:
You could call it getAndPrintHostname()?